www/docs/interests/category.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+// Logic page for the register of interests by category view
+// if no selected category, will just show a list of the categories
+
+include_once '../../includes/easyparliament/init.php';
+
+$chamber = get_http_var("chamber", "house-of-commons");
+$date = get_http_var("date", null);
```

Comment:
I might be missing something but there doesn't seem to be an interface to this?

Reply:
There isn't - it felt like a bolt on later and a bit of a tool for our purposes now.

Reply:
was just checking it wasn't a missing thing.